news_items/005_fractions_and_input_changes/index.html

```diff
@@ -0,0 +1,81 @@
+
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+</style>
+
+<p>
+  <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS '21)
+  has added some big improvements to PrairieLearn.
+</p>
+
+<h5>Fraction input</h5>
+
+<p>
+  One of the most-requested features ever! You can now enter fractions
```

Comment:
```suggestion
  One of the most-requested features ever! You can now enter fractions in [`pl-number-input`](https://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element)
```

Reply:
This is a student-facing news item (unlike the earlier ones) so I'm inclined to avoid technical details.

Reply:
How about:

> in elements where decimal numbers are supported?

Reply:
I thought "instead of numbers" would cover where fractions can be used? There are lots of details, but it seems too much for a student news item?

Reply:
I'm worried someone will be like:

Hey `pl-integer-input` accepts a rational fractions now! (It doesn't.) 

Also, this points to faculty as well where this is permissible... Double-edge sword on this news release.

Reply:
How about 'instead of decimal numbers'

Reply:
Does it help? Integers are decimal numbers too. I'm really not sure how to say this in a way that is both precise and clear. Maybe we should say "instead of numbers in most inputs"?

Reply:
This is one of those minor nits... Let's skip it. It's clear enough. Other things need to be worked on.